adoc/TRD-wordpress-on-lamp-on-sles.adoc

```diff
@@ -15,7 +15,7 @@
 The https://en.wikipedia.org/wiki/LAMP_(software_bundle)[LAMP] stack, while simple, is *ubiquitous*. 
 Its popularity in demos and tutorials all over the Internet to introduce a new concept or explain a complex one is like no other. 
 This guide details how to install and configure https://mariadb.org/[MariaDB], https://httpd.apache.org/[Apache], https://www.php.net/[PHP] on https://www.suse.com/products/server/[{sles}] to compose the https://en.wikipedia.org/wiki/LAMP_(software_bundle)[LAMP] stack. 
-To demonstrate our LAMP stack is functional, we will install and configure https://wordpress.com[WordPress] to use the stack. 
+To demonstrate your LAMP stack is functional, you will install and configure https://wordpress.com[WordPress] to use the stack. 
 The audience is expected to have a basic familiarity of Linux and be comfortable using the command line interface. 
 The result is not intended for production scenarios but rather for *workshops* or *demos*.
```

Comment:
Having a consistency regarding the use of "you" (this is how we usually approach the reader for direct instructions, steps, procedures) and "we" or neutral speech is of course good and right. However, this paragraph is an introductory paragraph, which gives an overview of the next "actions". Thus it would be appropriate to use "we" - above all as in the next sentence follows a sentence with indirect speech: "The audience is expected ...". To make the overview consistent, my suggestion is to change it to the following: 

```suggestion
To demonstrate the LAMP stack is functional, in this section https://wordpress.com[WordPress] will be installed and configured to use the stack. The audience is expected to have a basic familiarity of Linux and be comfortable using the command line interface.
```